### PR TITLE
Update Sass to 12.x, Mix to 6.0.x, remove dependencies we weren't using.

### PR DESCRIPTION
**Pull request by charlesstrange2**
https://trello.com/c/e4yunEIS/1749-admin-store-package-updates-vvv-e

### CODE REVIEW

**Review by neonleann — APPROVED** [review]
Let's do it! 😰

File: package.json [hunk 1]
```diff
@@ -2,34 +2,20 @@
...
     "dev": "npm run development",
-    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch-poll": "npm run watch -- --watch-poll",
-    "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "development": "mix",
+    "watch": "mix watch",
+    "watch-poll": "mix watch -- --watch-options-poll=1000",
+    "hot": "mix watch --hot",
     "prod": "npm run production",
```
Comment (Nikomus) [inline package.json:9]: Should these shortcuts be `yarn production` rather than npm?

Comment (charlesstrange2) [reply package.json:9]: 🤦‍♂️

Comment (charlesstrange2) [reply package.json:9]: 🤦 

though, wierdly it seems to have build 🤔 

i will correct.

**Pull request merged**